plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQueryClient.java

```diff
@@ -35,11 +35,13 @@
 import com.google.cloud.http.BaseHttpServiceException;
```

Comment:
Should we have something similar to https://github.com/trinodb/trino/pull/14500? Attaching unconditionally query id might be pointless to many use cases.

Reply:
We can make it session-configurable if you like. But I don't see a reason that it couldn't be added to every query job (it's just a label)